kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,20 @@
     });
   }
 
+  var draftbutton = $('.btn-draft');
+  var url = $('.btn-draft' ).data( 'draft-url' );
+  var draftmessage = $('.draft-message').html();
```

Comment:
When you're declaring a lot of variables all at once, you can just specify `var` once in the beginning, and put each new variable on a new line separated by commas. You can see an example of this [here](https://github.com/mozilla/kitsune/blob/master/kitsune/sumo/static/sumo/js/wiki.js#L125-L131)

It's also a good convention to prefix any variable names with `$` if they represent a jQuery-selected DOM element. So in this block of code you might want to change `draftbutton` to `$draftButton`.